src/package/package.ts

```diff
@@ -95,8 +97,15 @@ export class Package extends AsyncCreatable<PackageOptions> implements IPackage
     return Promise.resolve(undefined);
   }
 
-  public update(): Promise<void> {
-    return Promise.resolve(undefined);
+  public async update(options: PackageUpdateOptions): Promise<PackageSaveResult> {
+    // filter out any undefined values and their keys
+    Object.keys(options).forEach((key) => options[key] === undefined && delete options[key]);
```

Comment:
jsyk, this mutates the `options` object passed in, which some would argue is an anti-pattern.  It's probably fine but an alternative would be to use the filter function and assign to a new const that is used in the tooling update call.